Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM sitespeedio/webbrowsers:chrome-124.0-firefox-125.0-edge-123.0
+FROM sitespeedio/sitespeed.io:33.6.0
```

Comment:
@krompaco handle_update_docker method in tools/verify_result.py has to be modified also.

Reply:
I've made an attempt... Seems to work: https://github.com/krompaco/webperf_core/actions/runs/9066534196/job/24909642610#step:6:15